utils/cacharr_settings.py

```
@@ -0,0 +1,89 @@
+
+
+def _discover_prowlarr() -> tuple[str, str]:
+    """Return (api_key, base_url) from the first enabled Prowlarr instance.
+
+    Reads the API key from the instance's config.xml and constructs the
+    base URL from the configured port (default 9696).  Returns ("", "")
+    when no enabled instance with a readable key is found.
+    """
+    prowlarr_cfg = CONFIG_MANAGER.get("prowlarr") or {}
+    instances = prowlarr_cfg.get("instances") or {}
+    for inst_key, inst in instances.items():
```

Ruff (0.15.6)

[warning] Loop control variable `inst_key` not used within loop body

Rename unused `inst_key` to `_inst_key`

(B007)